data/hai/unfettered 0 prologue.txt

```diff
@@ -391,11 +391,11 @@ mission "Unfettered Jump Drive Trading"
 					to display
 						not "unfettered: uninterested in weapons"
 					goto uninterested
-				`	(Leave and come back another time.)`
+				`	"I'm not really interested in this anymore. I'm leaving."`
```

Comment:
I'm not sure this is a typo fix. I'd ask Hurl about it.

Reply:
It doesn't make sense that the Unfettered Hai knows that you're leaving, without telling them.

Reply:
```suggestion
				`	"I'm not really interested in this anymore. I'm leaving, for now."`
```